### ISSUE TITLE
Training not convergence on small dataset

### ISSUE THREAD

**Issue by cyiano** [hustvl/DiffusionDrive#48]
Hi, thanks for your excellent work. I want to train this repo, on a very small navsim trainset (~1k samples, randomly picked) for ovefitting test. But I found the loss does not converge even after 7000 iterations (as the tensorboard shows), the other configs remain the same as this repo. Is it normal on diffusion model training? Pls let me know If I do something wrong, thanks!

<img width="1187" alt="Image" src="https://github.com/user-attachments/assets/36d81581-c387-47b7-b749-55f1d5af8bdb" />

<img width="1375" alt="Image" src="https://github.com/user-attachments/assets/79fb5d44-61a8-4ded-95c8-d4f1c174ce37" />

**Comment by LegendBC** [comment]
Hi @cyiano , thanks for your interest in our work!

I haven't tried training DiffusionDrive on such a small dataset myself. You might want to tune some hyperparameters (e.g., learning rate, batch size, or number of timesteps) to see if that helps with convergence.

Thanks!
